content/tutorials/mushr_unity_sim.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using unity to simulate MuSHR"
+date: 2021-05-17T17:51:18+05:30
+summary: "Using a unity-based simulator in place of the default state prediction system that comes with mushr_base"
```

Comment:
Using a unity-based physics simulator in place of the default kinematic mushr_sim

Reply:
yeah, I did feel it was a bit wordy.